### ISSUE TITLE
Danger Express

### ISSUE THREAD

**Issue by arcadez2003** [libretro/mame2003-plus-libretro#1133]
Well look what has turned up...

https://github.com/mamedev/mame/commit/dcc41d25707a7d22c0c1d9b90da85785e0d4bec9

Bit of a back story here this was added to MAME quite some time ago then quickly removed due to some
hoarder complaining about it, i dunno if this is the same version or a different dump has turned up, either
way it's supported for the moment and here is my current WIP attempts to backport it to our ancient atarig42 driver.........

[dangerex.zip](https://github.com/libretro/mame2003-plus-libretro/files/7027609/dangerex.zip)

I haven't been able to test this yet as i'll likely not have the rom until tomorrow so i thought i'd post this up
here incase any of you guys manage to find it before me, there are a couple of things im concerned about
code wise but no point rabbiting on about it until we see if the game will work first.

**Comment by mahoneyt944** [comment]
I couldn't locate a rom to test.

**Comment by arcadez2003** [comment]
> 
> 
> I couldn't locate a rom to test.

Yeah same here it hasn't been made public yet hopefully it will be sometime this week

**Comment by arcadez2003** [comment]
@mahoneyt944 Here ya go i think everything is good, the game required the Guardians Of The Hood ASIC65
and after that to fix missing sounds i used the rom table boot values as per Road Riot as our ole driver would expect
as those are totally diff in latest MAME.

[dangerex.zip](https://github.com/libretro/mame2003-plus-libretro/files/7048296/dangerex.zip)

Danger Express is working in MAME84 it should do here with the code i've sent but ya never know PS for
testing purposes the romeset is included.

Regards
